### PR TITLE
[T-6390][16.0][FIX]sale_stock_deposit: location constrain

### PR DESCRIPTION
**Pull request by Tisho99**
The error says: "There cannot be two Deposits with the same Partner and Warehouse"

Now the domain does what it is said. 

The company is for multicompany compatibility.

### CODE REVIEW

**Comment by Jaimermaccione** [comment]
@Tisho99 LGTM! Functional review in a local environment. It allows installation, and I have tested it with the creation of locations. It permits the creation of a standard location under WH, allows the creation of a deposit location, and does not allow the creation of two deposit locations with the same customer and warehouse.

File: sale_stock_deposit/__manifest__.py [hunk 1]
```diff
@@ -7,6 +7,7 @@
     "version": "16.0.1.1.1",
```
Comment (manuelregidor) [inline sale_stock_deposit/__manifest__.py]: The version number needs to be changed

File: sale_stock_deposit/models/sale_order.py [hunk 2]
```diff
@@ -64,12 +60,13 @@ def action_confirm(self):
...
                 raise UserError(
                     _(
-                        "Route's warehouse '{}' for product '{}' is not the same"
-                        " as warehouse '{}' selected for the sale order."
-                    ).format(
-                        line.route_id.warehouse_id.display_name,
-                        line.product_id.display_name,
-                        line.warehouse_id.display_name,
+                        "Route's warehouse "
+                        "'{line.route_id.warehouse_id.display_name}' "
```
Comment (manuelregidor) [inline sale_stock_deposit/models/sale_order.py]: Isn't the f missing? It applies to this line and lines 66 and 68

**Review by manuelregidor — CHANGES_REQUESTED** [review]

**Review by manuelregidor — APPROVED** [review]
Technical review LGTM

**Pull request merged**